src/Sign.Cli/TemporaryConsoleEncoding.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE.txt file in the project root for more information.
+
+using System.Text;
+
+namespace Sign.Cli
+{
+    internal sealed class TemporaryConsoleEncoding : IDisposable
+    {
+        private readonly Encoding _defaultInputEncoding;
+        private readonly Encoding _defaultOutputEncoding;
+
+        internal TemporaryConsoleEncoding()
+        {
+            _defaultInputEncoding = Console.InputEncoding;
+            _defaultOutputEncoding = Console.OutputEncoding;
+
+            Console.InputEncoding = Encoding.UTF8;
```

Comment:
Should we have escape hatch something like env var which default to previous experience just in case?
Encoding thing could be tricky.